src/tests/Microsoft.PowerFx.Interpreter.Tests/InterpreterExpressionTestCases/Disabled.txt

```diff
@@ -1,3 +1,7 @@
 ﻿// List files that aren't supported in interpreter.
 #disable: Weekday.txt
 #disable: TrigTableFunctions.txt
+#disable: Abs_ConsistentOneColumnTableResult.txt
+#disable: Exp_ConsistentOneColumnTableResult.txt
+#disable: Int_ConsistentOneColumnTableResult.txt
+#disable: Ln_ConsistentOneColumnTableResult.txt
```

Comment:
Why disabled? We should have some test for this feature

Reply:
These table arg overloads for these functions aren't implemented in PowerFx Interpreter.

Reply:
In this case let's implement at least one of them so that we have coverage for these changes...